app/src/main/java/com/glia/exampleapp/Utils.java

```diff
@@ -115,7 +109,6 @@ public static UiTheme getRunTimeThemeByPrefs(SharedPreferences sharedPreferences
 
         UiTheme.UiThemeBuilder builder = new UiTheme.UiThemeBuilder();
 
-        builder.setAppBarTitle(title);
```

Comment:
Just out of interest, what is the reason for removing `setAppBarTitle()`?

Reply:
This property internally was not in use, so had no effect for a while, that is why I removed it.
Regarding release notes and migration guides, relying on the fact that we made activities internal and introduced EngagementLauncher instead should fully cover the migration part.